.github/workflows/handle-pr-open.yml

```diff
@@ -0,0 +1,29 @@
+name: Handle PRs
+
+on:
+  pull_request_target:
+    types: [opened]
+
+permissions:
+  contents: read
+  pull-requests: write
+  issues: write
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        name: Checkout default branch
+      - uses: actions/github-script@v3
```

Comment:
Any specific reason to use v3?
The latest major is 7

Reply:
No reason, I was just on the wrong branch tag when I was checking the version. I'll replace it with SHA values as well.